### PR TITLE
[Feature] Add InternalServerWrapper to dist

### PR DESCRIPTION
**Pull request by brapoprod**
This PR adds the InternalServerWrapper to the normal dist folder, so that it gets exposed in UNPKG as well

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@chili-publish/publisher-connector",
-  "version": "0.1.1",
+  "version": "0.1.0",
```
Comment (Storm-BE) [inline package.json]: Was it your intention to downgrade the version here?

Comment (brapoprod) [reply package.json]: Seems like you're viewing an old commit, if you check out all commits you'll see that the opposite has been applied and the version is bumped to 0.1.2

Comment (brapoprod) [reply package.json]: https://github.com/chili-publish/publisher-connector/pull/4/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519

Comment (Storm-BE) [reply package.json]: Yep, I was watching a single commit :) Thx!

**Review by Storm-BE — APPROVED** [review]

**Comment by seancrowe** [comment]
@brapoprod as we chatted, I made a pull request to your branch - see #5 

Merge into this, and then I will merge this into main.

**Review by seancrowe — APPROVED** [review]

**Pull request merged**